metrics/sources/kubelet/kubelet.go

```diff
@@ -132,25 +157,12 @@ func (this *kubeletMetricsSource) decodeMetrics(c *cadvisor.ContainerInfo) (stri
 			}
 		}
 
+		// No Kubernetes metadata so treat this as a system container.
 		if cName == "" || ns == "" || podName == "" {
```

Comment:
Hmm I thought all containers in pods were either explicitly named in pod spec or assigned a name when created. Is that not right?